tests/config/mod.rs

```diff
@@ -171,6 +171,7 @@ fn test_serde_custom_tikv_config() {
         compaction_readahead_size: ReadableSize::kb(1),
         info_log_max_size: ReadableSize::kb(1),
         info_log_roll_time: ReadableDuration::secs(12),
+        info_log_keep_log_file_num: 1000,
```

Comment:
You may want to use a different value than the default for the test.

Reply:
ok